src/app/containers/App/App.test.jsx

```diff
@@ -14,7 +14,7 @@ describe('App', () => {
   let wrapper;
   const initialData = { pageData: 'Some initial data' };
   const error = 'Error!';
-  const match = { params: { service: 'news', amp: false } };
+  const match = { params: { service: 'news', amp: false, variant: '/simp' } };
```

Comment:
```suggestion
  const match = { params: { service: 'news', amp: false, variant: 'simp' } };
```

Reply:
This is needed :) `/simp` is what we get back from the match :)

Reply:
Confirmed `/simp` is returned from the match. Confused this with the sanitised variant.